src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -413,6 +416,8 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                             source: $source,
                             image: $runtime['image'],
                             version: $version,
+                            cpus: $spec['cpus'] ?? 1,
+                            memory: $spec['memory'] ?? 512,
```

Comment:
We shouldn't limit build container to this. We either need ability to configure different specs for build, or we need some high build imits hard-coded.

Reply:
Not sure what to do here, make 1024 the minimum or make s-1vcpu-1gb the minimum specification?